.gitignore

```diff
@@ -4,3 +4,4 @@ dist
 dist-ssr
 *.local
 *.tgz
+.vscode
```

Comment:
I see a little warning here that there is no new line at the end of this file. 
Can you run `npm install` at the root of this project, then save this file again.
I'm thinking the pre-commit tool didn't run on your local machine, therefor the projects prettier config didn't run

Reply:
Not sure what's going on with my vscode and this repo but I fixed all the formatting issues.